src/main/java/edu/umich/ctools/sectionsUtilityTool/Friend.java

```diff
@@ -280,8 +258,8 @@ public CreateAccountResponse doSendInvite(String accountEmail,
 			map.put(INSTRUCTOR_NAME_TAG, instructorName);
 			map.put(CONTACT_EMAIL_TAG, contactEmail);
 
-			emailMessage = Friend.readFile(friendEmailFile, StandardCharsets.UTF_8);
-			emailMessage = replacePlaceHolders(emailMessage, map);
+			emailMessage = Utils.readFile(friendEmailFile, StandardCharsets.UTF_8);
```

Comment:
Since it is repeated you could make these couple of lines a method and just pass in the file name and maybe the map.  The method would be small but the purpose would be clearer, there would be less code to read, and less to figure out about what is happening.